src/ResourceOperations/ResourceHolder.cs

```diff
@@ -206,8 +208,32 @@ private void UpdateFile(string filename, string valueColumnId, bool skipNontrans
             }
 
             // Write the cached resources to the drive
-            using (var writer = new ResXResourceWriter(filename))
+            using (var writer = new ResXResourceWriter(filename))//$"{filename.Replace(".resx","")}_.resx"))

```

Comment:
Make sure to remove commented out code or unnecessary comments before submitting a PR, please! :)

Reply:
OK